### PR TITLE
Updated json and .md file as per the code

### PR DESCRIPTION
**Pull request by svenka471**
Documentation enhancement for UsbAccess Plugin.
jira : https://ccp.sys.comcast.net/browse/RDK-34924

### CODE REVIEW

File: UsbAccess/UsbAccess.json [hunk 1]
```diff
@@ -26,17 +26,12 @@
...
         "clearLink": {
-            "summary": "Clears or removes the symbolic link created by the `createLink` method",
+            "summary": "Clears or removes the symbolic link created by the `createLink` method. On failure, response returns error message and status of cleared link.\n \n### Events \n\n No Events.",
             "result": {
                 "type": "object",
                 "properties": {
                     "success":{
                         "$ref": "#/definitions/success"
-                    },
-                    "error":{
```
Comment (npoltorapavlo) [inline UsbAccess/UsbAccess.json:36]: why you are removing the "error" property from all responses?

Comment (svenka471) [reply UsbAccess/UsbAccess.json]: Updated the changes as per your  review comments.

Comment (npoltorapavlo) [reply UsbAccess/UsbAccess.json]: looks good

File: UsbAccess/doc/UsbAccessPlugin.md [hunk 2]
```diff
@@ -368,14 +401,61 @@ This method takes no parameters.
...
         "success": true,
         "error": "invalid filename"
     }
 }
 ```
 
+<a name="method.ArchiveLogs"></a>
+## *ArchiveLogs [<sup>method</sup>](#head.Methods)*
+
+(Version 2) Creates a log directory and archives the logs files of the device.
```
Comment (anand-ky) [inline UsbAccess/doc/UsbAccessPlugin.md:415]: Pls change description to indicate that logs will be uploaded to USB. 
Something like "Compresses and uploads device logs into attached USB drive"

Can we also give details on which location the logs will go into.

**Review by anand-ky — CHANGES_REQUESTED** [review]

File: UsbAccess/doc/UsbAccessPlugin.md (hunk 2 above)
Comment (svenka471) [reply UsbAccess/doc/UsbAccessPlugin.md]: Updated the changes as per your review comments.

**Review by anand-ky — APPROVED** [review]

**Pull request merged**